scss/login.scss

```
@@ -0,0 +1,69 @@
+.background-login {
+  align-items: center;
+  background-color: $white-secondary;
+  display: flex;
+  flex-direction: column;
+  height: 768px;
+  justify-content: center;
+  width: 1366px;
+}
+
+.container-login {
+	height: 400px;
+  width: 300px;
+  background-color: $white-primary;
+	border-top: 8px solid $blue;
+	display: flex;
+	flex-direction: column;
+
+	.login-logo {
+		height: 70px;
+		width: 212px;
+		margin: 20px 44px 0 44px;
+	}
+	
+	label {
+		color: $black-default;
+		font-size: 16px;
+		height: 22px;
+		line-height: 22px;
+		margin: 16px 24px 8px 24px;
```

This rule can be simplified with a shorthand
```suggestion
		margin: 16px 24px 8px;
```